--- d:\Aculab\v6\include\acu_type.h	2008-12-30 17:09:33.118929600 +0100
+++ acu_type.h2	2008-12-30 17:03:02.597387200 +0100
@@ -80,7 +80,7 @@
         #define TiNGTYPE_WINNT
     #endif
 
-    #define ACU_EXPORT WINAPI
+    #define ACU_EXPORT
 
     #ifndef DRIVER
 		/* Streamline what we get from windows.h */
@@ -161,7 +161,7 @@
        #define ACU_PACK_DIRECTIVE
    #endif /* ACU_SOLARIS_SPARC */
 #else
-   #define ACU_PACK_DIRECTIVE __attribute__ ((packed))
+   #define ACU_PACK_DIRECTIVE
 #endif /* ACU_LINUX */
 
 
