--- /usr/local/aculab/v6/include/cl_lib.h	2006-05-29 15:08:16.000000000 +0200
+++ cl_lib.h2	2006-07-11 13:32:01.000000000 +0200
@@ -16,16 +16,10 @@
 
 #include "acu_type.h"
 
-#ifndef ACU_LINUX
-#ifndef ACU_SOLARIS_SPARC
-# pragma pack (push, 1 )
-# define ACU_PACK_DIRECTIVE
-#else
-# define ACU_PACK_DIRECTIVE
-#endif /* ACU_SOLARIS_SPARC */
-#else
-# define ACU_PACK_DIRECTIVE __attribute__ ((packed))
-#endif /* ACU_LINUX */
+#define ACU_EXPORT
+#define ACU_PACK_DIRECTIVE
+#define ACU_DEPRECATE_PRE
+#define ACU_DEPRECATE_POST
 	
 enum
 {
